app/views/govuk_publishing_components/components/_list.html.erb

```diff
@@ -0,0 +1,26 @@
+<%
+  id ||= nil
+  extra_spacing ||= nil
+  list_style ||= nil
+  list_type ||= "unordered"
```

Comment:
I'm wondering if this would be more usable as a true/false flag, since there are only two options? Something like

```
bullets ||= false
...

classes << "govuk-list--bullet" if visible_counters && bullets
classes << "govuk-list--number" if visible_counters && bullets == false
...

list_tag = "ol" unless bullets
```

Reply:
Whilst there are only two options and a boolean would work, I think setting `list_type` to "ordered" or "unordered" gives a clearer intent and doesn't confuse with whether the counters are shown or not. 

It also gives us space to expand to include other ordinals - for example, square bullets or Roman numerals - without breaking backwards compatibility.

Reply:
Ah okay hadn't thought about other options like that. Just whenever I see strings passed as options it looks wrong to me.